### PR TITLE
Sentry tracking 

### PR DESCRIPTION
**Pull request by MikePeng06**

### CODE REVIEW

File: dev/.project [hunk 1]
```diff
@@ -27,12 +27,12 @@
 		<link>
 			<name>board</name>
 			<type>2</type>
-			<location>D:/Github/RMdevBoard_ChibiOS/hw</location>
```
Comment (TopQuark12) [inline dev/.project]: Don't stage personal environment preferences into a commit.

File: dev/.project [hunk 2]
```diff
@@ -27,12 +27,12 @@
...
+			<location>C:/Users/zpeng/OneDrive/Desktop/RoboMaster/Newest/RMdevBoard_ChibiOS/hw</location>
 		</link>
 		<link>
 			<name>os</name>
 			<type>2</type>
-			<location>D:/Github/RMdevBoard_ChibiOS/os</location>
```
Comment (TopQuark12) [inline dev/.project]: Don't stage personal environment preferences into a commit.

**Comment by MikePeng06** [comment]
changed to the default ones

**Comment by Beck-Sisyphus** [comment]
请push到sentry branch

**Pull request closed without merging**